blockbase/functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+if ( ! function_exists( 'blockbase_support' ) ) :
+	function blockbase_support()  {
+
+		// Alignwide and alignfull classes in the block editor
+		add_theme_support( 'align-wide' );
+
+		// Adding support for core block visual styles.
+		add_theme_support( 'wp-block-styles' );
+
+		// Adding support for responsive embedded content.
+		add_theme_support( 'responsive-embeds' );
+
+		// Add support for experimental link color control.
+		add_theme_support( 'experimental-link-color' );
+
+		// Add support for editor styles.
+		add_theme_support( 'editor-styles' );
+
+		// Enqueue editor styles.
+		add_editor_style( array( 
+			'style.css',
+			'assets/alignments-editor.css'
+		) );
+    }
+    add_action( 'after_setup_theme', 'blockbase_support' );
+endif;
+
+/**
+ * Enqueue scripts and styles.
+ */
+function blockbase_scripts() {
```

Comment:
Just thinking out loud... if we can eliminate the add_theme_support calls above, this file would only exist to enqueue the stylesheets. Would it be possible to enqueue the stylesheets from within `theme.json` instead?

Reply:
I've opened an issue here to discuss: https://github.com/WordPress/gutenberg/issues/26902